mappings/net/minecraft/container/DoubleContainer.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS agm net/minecraft/container/DoubleContainer
```

Comment:
I'm guessing this is used for double-chests? The `Double` was quite confusing to me at first; maybe something like `JoinedContainer`?